src/test/java/webdriver/DriverManager.java

```diff
@@ -0,0 +1,21 @@
+package webdriver;
+
+import org.openqa.selenium.WebDriver;
+
+public class DriverManager {
+    private static WebDriver driver = null;
+
+    public static WebDriver getDriver(){
+        return driver;
+    }
+
+    public static void setDriver (WebDriver webDriver) {
```

Comment:
if you call this method twice in code your driver will be rewritten, you need to exclude such situations